src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health, hidden) {
+    this.name = name;
+    this.health = health || 100;
+    this.hidden = hidden || false;
+  };
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name, health, hidden) {
+    super(name, health, hidden);
+    Animal.alive.push(this);
```

Comment:
you can push in animal constructor no to repeat code

Reply:
good point!!!